lib/entities/alerts/enums.ts

```diff
@@ -22,10 +22,12 @@ export enum AlertConditionType {
 
 export enum AlertActionType {
   User = "User",
+  Integration = "Integration",
```

Comment:
You will need to bump the SDK version inside the package.json if this is a separate deployment or if you want to merge into Crystals branch you won't need to bump the version in this branch :)

Reply:
Thanks for pointing this out! Have changed the base to Crystal's branch 👍